src/main/resources/db/migration/V3__DDL.sql

```diff
@@ -1,6 +1,7 @@
-CREATE VIEW eservice_view AS
-SELECT e.id, e.eservice_id , e.eservice_name, e.producer_name , e.version_id , e.state ,e.probing_enabled , e.version_number , epr.response_received
+CREATE VIEW ${schema_name}.eservice_view AS
+SELECT e.id, e.eservice_id , e.eservice_name, e.producer_name , e.version_id , e.state ,e.probing_enabled , e.version_number , epr.response_received, epreq.last_request, e.polling_frequency
```

Comment:
are you sure this query is valid? 
I see fields in the select that are not in the group by.

Plus, what is the purpose of the group by clause?

Reply:
at first instance it was included because there was ambiguity and the script would fail, I re-executed it without group by and it is now Ok.

Reply:
this is marked as resolved but the group by has not been removed